docs/src/pages/docs/api.md

```diff
@@ -151,6 +151,74 @@ const secondaryAxes = React.useMemo(
 )
 ```
 
+### Chart Component Props
+
+The Chart component props can be passed in its option property object:
+
+```javascript
+<Chart
+  options={{
+    data,
+    primaryAxis,
+    secondaryAxes,
+  }}
+/>
+```
+
+The data property should be an array of series, each series should be an array of objects.
+Each object should have two properties, by convention called primary and secondary, but it can be named as you want. One of these properties will be used as the primary axis and the other as the secondary axes.
+
+```javascript
+const data = [
+  {
+    label: 'Series 1',
+    data: [
+      {
+        primary: '2022-02-03T00:00:00.000Z',
+        likes: 130,
+      },
+      {
+        primary: '2022-03-03T00:00:00.000Z',
+        likes: 150,
+      },
+    ],
+  },
+  {
+    label: 'Series 2',
+    data: [
+      {
+        primary: '2022-04-03T00:00:00.000Z',
+        likes: 200,
+      },
+      {
+        primary: '2022-05-03T00:00:00.000Z',
+        likes: 250,
+      },
+    ],
+  },
+]
+```
+
+The **primaryAxis** and **secondaryAxes** props should be a function that returns and implements a getter function called getValue.
```

Comment:
Should have a prop called getValue, which is a getter function that returns the axis value for the datum.